src/common/quantile.h

```diff
@@ -408,7 +193,8 @@ struct WXQSummary : public WQSummary<DType, RType> {
     }
```

Comment:
> CHECK(nbig < n) << "quantile: too many large chunk";

I recently run into this error when weight is sampled from a Gaussian distribution.  Not sure what's happening here.